Azure/sonic-buildimage
# Add BRCM SOC Property to not count ACL drops towards interface RX_DRP…

… counter for 7050CX3 and 7260CX3 DualToR platforms

#### Why I did it
For DualToR There will be expected packet drops that should not be counted towards interface RX_DRP counters.
SAI support this feature but is only enabled if the proper SOC property is set in the device's conf.bcm file.
this change is to add the necessary SOC property for the HW SKUs that are purposed for DualToR Roles.

#### How to verify it
Send packet that is expected to be drop and check that it is not being counted towards interface RX_DRP counter.
Also tested with the case where  SAI change is not yet there but SOC is already enabled and observe it does not cause any undesirable side-effect.

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
Add SOC property to not count ACL Drops towards interface RX_DRP for DualToR role HW SKUs.

#### A picture of a cute animal (not mandatory but encouraged)
